stack-cmbm/files/default/bashrc

```diff
@@ -0,0 +1,102 @@
+# ~/.bashrc: executed by bash(1) for non-login shells.
```

Comment:
where is this used? Also, maybe use `/etc/bashrc.d` instead, so you dont risk overwriting